doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_fota_update.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+.. _ug_nrf54h20_suit_fota_update:
+
+FOTA updates on nRF54H Series devices
+#####################################
+
+|fota_upgrades_def|
+You can also use FOTA updates to replace the application.
+
+FOTA over Bluetooth Low Energy
+******************************
+
+FOTA updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth.
+The application acts as a GATT server and allows the connected Bluetooth Central device to perform a firmware update.
+To use FOTA over Bluetooth LE, samples must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`).
+
+Testing steps
+=============
+
+To perform a FOTA update, complete the following steps:
+
+.. ug_nrf54h20_suit_fota_updates_over_ble_nrfcdm_common_dfu_steps_start
+
+1. Generate the SUIT envelope by building your application with the FOTA support over Bluetooth Low Energy.
+   You can find the generated :file:`root.suit` envelope in the :file:`<build_dir>/DFU` directory.
+   Alternatively, you can use the generated :file:`dfu_suit.zip` package in the :file:`<build_dir>/zephyr` directory.
+#. Download the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package to your device.
```

Comment:
`to your mobile device running the DFU mobile app` (to be more precise)

Reply:
Alternatively, we could skip this step as the subsequent step already states that 
```
a. Ensure that you can access the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package from your phone or tablet.
```

Reply:
Correct point 2 but kept the statement from point a